mw-webapp/src/component/table/Table.tsx

```diff
@@ -29,6 +29,17 @@ interface TableData<T> {
   rowModel: RowModel<T>;
 }
 
+/**
+ * Data attributes for cypress testing
+ */
+interface dataCyTableContent {
+
+  /**
+   * Data attribute for cypress testing
+   */
+  dataCyTh?: string;
+}
+
```

Comment:
why do you need new interface for just one stiring value

Reply:
For other tests with table i'll need more attributes. So i created content interface. Leave it as it was? and then, if necessary, redo it?

Reply:
@tritonJS826, you're right, all other elements on the page are other components and there is no point in making an interface with attributes for the table.